src/seedu/addressbook/AddressBook.java

```diff
@@ -381,6 +392,8 @@ private static String executeCommand(String userInputString) {
             return executeClearAddressBook();
         case COMMAND_HELP_WORD:
             return getUsageInfoForAllCommands();
+        case COMMAND_SORT_WORD:
```

Comment:
good that you created a constant for this and made sure that the method name is clear